internal/provisioner/kops_provisioner_cluster_installation.go

```diff
@@ -193,6 +187,17 @@ func (provisioner *kopsCIAlpha) HibernateClusterInstallation(cluster *model.Clus
 	return nil
 }
 
+func configInstallationForHibernation(mattermost *mmv1alpha1.ClusterInstallation) {
+	// Hibernation is currently considered changing the Mattermost app deployment
+	// to 0 replicas in the pod. i.e. Scale down to no Mattermost apps running.
+	// The current way to do this is to set a negative replica count in the
+	// k8s custom resource. Custom ingress annotations are also used.
+	// TODO: enhance hibernation to include database and/or filestore.
+	mattermost.Spec.Replicas = 0
```

Comment:
If I am not mistaken for old CR we need to set it to -1. I believe there is a const that was used before `hibernationReplicaCount`.